docker/scripts/install-docker-compose-ubuntu.sh

```diff
@@ -0,0 +1,28 @@
+# Ask for the user password
+# Script only works if sudo caches the password for a few minutes
+sudo true
+
+# Install kernel extra's to enable docker aufs support
+# sudo apt-get -y install linux-image-extra-$(uname -r)
+
+# Add Docker PPA and install latest version
+# sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
+# sudo sh -c "echo deb https://get.docker.io/ubuntu docker main > /etc/apt/sources.list.d/docker.list"
+# sudo apt-get update
+# sudo apt-get install lxc-docker -y
+
+# Alternatively you can use the official docker install script
+wget -qO- https://get.docker.com/ | sh
```

Comment:
Whenever possible, it's better to install packages from repos. Isn't Docker on Ubuntu repos?

Reply:
Yes, there is docker on the Ubuntu repository, but it is not always up to date, so it is why I decided to use that from the official Docker repository. Is that okay?

Reply:
It's an acceptable approach, however there are 2 reason to go for the distro-provided packages:
* for consistency with the corresponding Fedora/CentOS scripts,
* from a supportability perspective, it's a bad thing not to use a fixed/known version. Keep in mind that Ceph Dashboard devels might be using now 3 different distros (CentOS, Fedora, and now Ubuntu). Using pinned versions is always a best practice if you have to support multiple environments. There's no way to know in advance if a new docker release will deprecate an existing feature and will render unusable our current docker-compose configuration.

Reply:
This makes sense. Thanks for the insight. Let me then implement it that way